VmAgent.Core/Dependencies/BasicAssetExtractor.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Microsoft.Azure.Gaming.VmAgent.Core.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.Azure.Gaming.VmAgent.Core.Dependencies
+{
+    public class BasicAssetExtractor
+    {
+        public static BasicAssetExtractor Instance = new BasicAssetExtractor();
+
+        private readonly ISystemOperations _systemOperations;
+        private readonly MultiLogger _logger;
+        private string _zipExtension;
```

Comment:
Do we need to keep file extension parameters in Globals? Since we moved these methods under VmAgent.Core from VmAgent, need to figure it out how to access Global parameters.

Reply:
I just added a comment about it :)